avm/res/machine-learning-services/workspace/tests/e2e/ai/dependencies.bicep

```diff
@@ -0,0 +1,54 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
```

Comment:
Again - do we need to make these alphabetical?

Reply:
same response as above :)